lib/sentry/event.ex

```diff
@@ -255,6 +257,14 @@ defmodule Sentry.Event do
       `Sentry.capture_message/2`. *Available since v10.1.0*.
       """
     ],
+    integration_meta: [
+      type: :map,
+      doc: """
+      A map to store metadata specific to the integration.
+      It allows cron integrations, such as the Oban integration,
+      to store job-related metadata. *Available since v10.1.0*.
+      """
```

Comment:
For now this can be `doc: false` instead since integrations are all internal to the Sentry SDK.